### PR TITLE
brought repo up to date, added application files, added instructions

### PR DESCRIPTION
**Pull request by kristinapathak**

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+vendor
+argus
```
Comment (kcajmagic) [inline .dockerignore]:
```suggestion
__PROJECT__
```

**Review by kcajmagic — CHANGES_REQUESTED** [review]
`deploy/Dockerfile*` is no longer needed

File: deploy/Dockerfile.local [hunk 2]
```diff
@@ -0,0 +1,26 @@
+FROM golang:alpine as builder
```
Comment (joe94) [inline deploy/Dockerfile.local]: If we're adding a Dockerfile.local, should we add back the makefile recipe for it?

Comment (kristinapathak) [reply deploy/Dockerfile.local]: Do we want a Dockerfile.local? I thought it was supposed to be here, but can delete it.

Comment (joe94) [reply deploy/Dockerfile.local]: I use it pretty often for e2e validation locally https://github.com/xmidt-org/argus/blob/v0.3.9/Makefile#L58 
I don't think a dedicated .local file is necessary though.

**Review by kcajmagic — APPROVED** [review]

**Review by joe94 — APPROVED** [review]

**Review by schmidtw — APPROVED** [review]

**Pull request merged**